content/bn/docs/concepts/security/_index.md

```diff
@@ -0,0 +1,129 @@
+ব্যবহারকারীদের দ্বারা উত্পন্ন ক্রিয়াকলাপ, কুবার্নিটিস API ব্যবহার করা অ্যাপ্লিকেশন এবং নিয়ন্ত্রণ প্লেন নিজস্ব
+ক্রিয়াকলাপগুলি অডিট করে।
+
+## ক্লাউড প্রদায়কের নিরাপত্তা
+
+{{% thirdparty-content vendor="true" %}}
+
+আপনি যদি আপনার নিজের হার্ডওয়্যার বা একটি বিভিন্ন ক্লাউড প্রদায়কে একটি কুবারনেটিস ক্লাস্টার চালাচ্ছেন,
+নিরাপত্তা সেরা অনুশীলনের জন্য আপনার নথি দেখুন।
+এখানে কিছু জনপ্রিয় ক্লাউড প্রদায়কদের নিরাপত্তা নথিগুলির লিঙ্ক:
+
+{{< table caption="ক্লাউড প্রদায়কের নিরাপত্তা" >}}
+
+IaaS প্রদায়ক        | লিঙ্ক |
+-------------------- | ------------ |
+আলিবাবা ক্লাউড | https://www.alibabacloud.com/trust-center |
+আমাজন ওয়েব সার্ভিস | https://aws.amazon.com/security |
+গুগল ক্লাউড প্ল্যাটফর্ম | https://cloud.google.com/security |
+হুয়াওয়ে ক্লাউড | https://www.huaweicloud.com/intl/en-us/securecenter/overallsafety |
+আইবিএম ক্লাউড | https://www.ibm.com/cloud/security |
+মাইক্রোসফট আজওর | https://docs.microsoft.com/en-us/azure/security/azure-security |
+অরাকেল ক্লাউড ইন্ফ্রাস্ট্রাকচার | https://www.oracle.com/security |
+VMware vSphere | https://www.vmware.com/security/hardening-guides |
+
+{{< /table >}}
+
+## নীতি
+
+আপনি কুবারনেটিস-নেটিভ মেকানিজম ব্যবহার করে নিরাপত্তা নীতি নির্ধারণ করতে পারেন,
+যেমন [NetworkPolicy](/docs/concepts/services-networking/network-policies/)
+(নেটওয়ার্ক প্যাকেট ফিল্টারিং উপর ঘোষণামূলক নিয়ন্ত্রণ) বা
+[ValidatingAdmisisonPolicy](/docs/reference/access-authn-authz/validating-admission-policy/) (কে কি 
+পরিবর্তন করতে পারে তা সম্পর্কে উঘোষণামূলক সীমাবদ্ধতা).
+
+তবে, আপনি কুবারনেটিস পরিবেশের চারপাশে নীতি কার্যান্বয়নে নির্ভর করতে পারেন। কুবার্নিটিস এক্সটেনশন মেকানিজম সরবরাহ করে
+এই পরিবেশ প্রকল্পগুলির উপর তাদের নিজস্ব নীতি নিয়ন্ত্রণ সাধারণের জন্য
+উন্মোচনের সুযোগ প্রদান করতে। এগুলি উদাহরণ হিসেবে উল্লেখ করা যেতে পারে:
+সোর্স কোড পর্যালোচনা, কন্টেনার ইমেজ অনুমোদন, এপিআই অ্যাক্সেস নিয়ন্ত্রণ,
+নেটওয়ার্কিং, এবং অন্যান্য।
+
+নীতি মেকানিজম এবং কুবারনেটিসের সম্পর্কে আরও তথ্য জানতে,
+[নীতি](/docs/concepts/policy/) পড়ুন।
+
+## {{% heading "whatsnext" %}}
+
+সম্পর্কিত কুবারনেটিস নিরাপত্তা বিষয়গুলি জানুন:
+
+* [আপনার ক্লাস্টার নিরাপত্তা সুরক্ষা করা](/docs/tasks/administer-cluster/securing-a-cluster/)
+* [পরিচিত দুর্বলতা](/docs/reference/issues-security/official-cve-feed/)
+  কুবারনেটিসে (এবং আরও তথ্যের লিঙ্ক)
+* [ট্রানজিটে ডেটা এনক্রিপশন](/docs/tasks/tls/managing-tls-in-a-cluster/) কন্ট্রোল প্লেনের জন্য
+* [অবশিষ্ট ডেটা এনক্রিপশন](/docs/tasks/administer-cluster/encrypt-data/)
+* [কুবারনেটিস API অ্যাক্সেস নিয়ন্ত্রণ](/docs/concepts/security/controlling-access)
+* [নেটওয়ার্ক নীতি](/docs/concepts/services-networking/network-policies/)
+* [কুবার্নিটিসে গোপনীয়তা](/docs/concepts/configuration/secret/)
+* [পডগুলির নিরাপত্তা মান](/docs/concepts/security/pod-security-standards/)
+* [রানটাইমক্লাস](/docs/concepts/containers/runtime-class)
+
+প্রসঙ্গ জানুন:
+
+<!-- এটি পরিবর্তন করলে, content/en/docs/concepts/security/cloud-native-security.md এর ফ্রন্ট ম্যাটার ম্যাচ করতে অন্যত্র সম্পাদনা করুন; no_list সেটিং পরীক্ষা করুন -->
```

Comment:
you don't have to localize comment



<!-- if changing this, also edit the front matter of content/en/docs/concepts/security/cloud-native-security.md to match; check the no_list setting -->